Mathlib/Analysis/Complex/Hadamard.lean

```diff
@@ -421,5 +426,229 @@ lemma norm_le_interp_of_mem_verticalClosedStrip' (f : ℂ → E) {z : ℂ} {a b
+  to the strip ``re ⁻¹' (0,1)``-/
+lemma scale_mapsto_dom {l u : ℝ} (hul: l<u) {z: ℂ} (hz: z ∈ verticalStrip 0 1) :
+    l + z * (u-l)  ∈ verticalStrip l u := by
+  simp only [verticalStrip, mem_preimage, mem_Ioo] at hz
+  simp only [verticalStrip, mem_preimage, add_re, ofReal_re, mul_re, sub_re, sub_im, ofReal_im,
+    sub_self, mul_zero, sub_zero, mem_Ioo, lt_add_iff_pos_right]
+  obtain ⟨hz₁, hz₂⟩ := hz
+  simp only [hz₁, mul_pos_iff_of_pos_left, sub_pos, hul, true_and]
+  rw [add_comm, ← sub_lt_sub_iff_right l, add_sub_assoc, sub_self, add_zero]
+  nth_rewrite 2 [← one_mul (u-l)]
+  gcongr
+  simp only [sub_pos]
+  exact hul
+
+
+/--The transformation on ℂ that is used for `scale` maps the closed strip ``re ⁻¹' [l,u]``
+  to the closed strip ``re ⁻¹' [0,1]``-/
+lemma scale_mapsto_dom_cl {l u : ℝ} (hul: l<u) {z: ℂ} (hz: z ∈ verticalClosedStrip 0 1) :
+    l + z * (u-l)  ∈ verticalClosedStrip l u := by
+  simp only [verticalClosedStrip, mem_preimage, add_re, ofReal_re, mul_re, sub_re, sub_im,
+    ofReal_im, sub_self, mul_zero, sub_zero, mem_Icc, le_add_iff_nonneg_right]
+  simp only [verticalClosedStrip, mem_preimage, mem_Icc] at hz
+  obtain ⟨hz₁, hz₂⟩ := hz
+  simp only [sub_pos, hul, mul_nonneg_iff_of_pos_right, hz₁, true_and]
+  rw [add_comm, ← sub_le_sub_iff_right l, add_sub_assoc, sub_self, add_zero]
+  nth_rewrite 2 [← one_mul (u-l)]
+  gcongr
+  simp only [sub_nonneg]
+  exact le_of_lt hul
+
+
+/--If z is on the closed strip `re ⁻¹' [l,u]`, then `(z-l)/(u-l)` is on the closed strip
+  `re ⁻¹' [0,1]`.-/
+lemma scale_mem_strip {z : ℂ} {l u : ℝ} (hul: l < u) (hz: z ∈ verticalClosedStrip l u) :
+    z/(u - l) - l/(u-l) ∈ verticalClosedStrip 0 1 := by
+  simp only [verticalClosedStrip, Complex.div_re, mem_preimage, sub_re, mem_Icc,
+    sub_nonneg, tsub_le_iff_right, ofReal_re, ofReal_im, sub_im, sub_self, mul_zero, zero_div,
+    add_zero]
+  simp only [verticalClosedStrip] at hz
+  norm_cast
+  simp_rw [Complex.normSq_ofReal, mul_div_assoc, div_mul_eq_div_div_swap,
+    div_self (by linarith : u - l ≠ 0), ← div_eq_mul_one_div]
+  constructor
+  · gcongr
+    · apply le_of_lt; simp [hul]
+    · exact hz.1
+  · rw [← sub_le_sub_iff_right (l / (u-l)), add_sub_assoc, sub_self, add_zero, div_sub_div_same,
+      div_le_one (by simp[hul]), sub_le_sub_iff_right l]
+    exact hz.2
+
+
+/-- The function `scale f l u` is `diffContOnCl`. -/
+lemma scale_diffContOnCl [NormedSpace ℂ E] {f: ℂ → E} {l u : ℝ} (hul: l < u)
+    (hd : DiffContOnCl ℂ f (verticalStrip l u)) : DiffContOnCl ℂ (scale f l u)
+    (verticalStrip 0 1) := by
+  unfold scale
+  apply DiffContOnCl.comp (s:= verticalStrip l u) hd
+  · apply DiffContOnCl.const_add
+    apply DiffContOnCl.smul_const
+    exact Differentiable.diffContOnCl differentiable_id'
+  · rw [MapsTo]
+    intro z hz
+    exact scale_mapsto_dom hul hz
+
+
+/-- The norm of the function `scale f l u` is bounded above on the closed strip `re⁻¹' [0, 1]`-/
+lemma scale_bddAbove {f: ℂ → E} {l u : ℝ} (hul: l< u)
```

Comment:
```suggestion
/-- The norm of the function `scale f l u` is bounded above on the closed strip `re⁻¹' [0, 1]`-/
lemma scale_bddAbove {f: ℂ → E} {l u : ℝ} (hul: l < u)
```
Can you standardize the spacing in the file? Normally things like `u-l` should really be written as `u - l` and so on.